src/tasks/AotCompilerTask/MonoAOTCompiler.csproj

```
@@ -11,6 +11,9 @@
     <NoWarn Condition="$(TargetFramework.StartsWith('net4'))">$(NoWarn),CS8604,CS8602</NoWarn>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+    <ProjectReference Include="$(RepoRoot)src\coreclr\tools\dotnet-pgo\dotnet-pgo.csproj" />
```

This is just for making sure the tool gets built, correct? if so:
```suggestion
    <ProjectReference Include="$(RepoRoot)src\coreclr\tools\dotnet-pgo\dotnet-pgo.csproj" ReferenceOutputAssembly="false" />
```

---

Oh right, I'm actually undoing this change. There was some offline discussion on whether or not we are bundling dotnet-pgo with the MonoAOTCompiler, but there is a possibilty of dotnet-pgo being shipped as a global tool, so we are opting to have users install the tool and just have the MonoAOTCompiler Task call the executable from that path instead. I think @steveisok has some changes related to this.